PantryPal/HomeScreen.js

```diff
@@ -18,7 +18,10 @@ const HomeScreen = ({ navigation }) => {
       style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}
     >
       <View style={styles.container}>
-        <Text style={styles.title}> Pantry Pal </Text>
+        <Image
+          source={require('./Images/pantrypal.jpg')}
```

Comment:
Cannot use require. It causes the whole app to not function. To resolve the issue, you should 
import image from './Images/pantrypal.jpg';
and then set source={image} in order to prevent the app from not functioning.